packages/flutter/lib/src/rendering/object.dart

```diff
@@ -2857,6 +2857,11 @@ abstract class RenderObject with DiagnosticableTreeMixin implements HitTestTarge
   /// parentUsesSize ensures that this render object will undergo layout if the
   /// child undergoes layout. Otherwise, the child can change its layout
   /// information without informing this render object.
+  ///
+  /// It is recommended to establish / update the paint transform of this render
```

Comment:
I assume this means you want `applyPaintTransform` to return the correct answer after layout? Maybe link to that method from here so people have context on what the "paint transform" is?

Reply:
Maybe also have it link to getTransformTo for even more context.

Reply:
Rephrased the paragraph. Note to myself to change the backticks to square brackets in PR2 for the `OverlayPortal.overlayChildLayoutBuilder` reference.